### DISCUSSION TITLE
White Paper

### DISCUSSION THREAD

**Discussion by paulmueller** [AFM-SPM/home#6] [General]
https://en.wikipedia.org/wiki/White_paper

We should write a white paper, stating what we are actually up to here (in this organization) and in general (as a community). I guess for a start we can collect ideas in this discussion.

**Comment by paulmueller** [comment]
It should be written for AFM newbies, e.g. for PhD students who need orientation.

**Comment by bobturneruk** [comment]
Sounds good. I propose we start with a `readme.md` in the home repository and build from there?
